docs/development/getting_started.rst

```diff
@@ -54,24 +54,34 @@ Either way, once you've installed Vagrant you should run:
 
 Finally, install Ansible so it can be used with Vagrant to automatically
 provision VMs. We recommend installing Ansible from PyPi with ``pip`` to ensure
-you have the latest stable version.
+you have the latest stable version:
 
 .. code:: sh
 
     sudo apt-get install python-pip
-    sudo pip install -U ansible
 
-If you're using Ubuntu, you can install a sufficiently recent version of
-Ansible from backports (if you prefer): ``sudo apt-get install
-ansible/trusty-backports``
+Since the version of ansible required to provision SecureDrop VMs may flux out
+of sync with the version in your distro's repos, we recommend installing the
+latest stable ansible to a virtual environment using `virtualenvwrapper` or
```

Comment:
Just updated then installed ansible on Tails 2.3, the latest stable, and they are shipping 1.7.2. Talking with Conor, he said he just installed SD on real hw using this version. It makes sense to test our playbooks against the version one would be using to do a real install, so I am going to rebase with that version and test it on my own workstation to make sure it works.